.travis.yml

```diff
@@ -3,3 +3,5 @@ python: 3.5
 cache: pip
 before_install: pip install -U pre-commit
 script: pre-commit run --all-files
+notifications:
+  email: false
```

Comment:
Couldn't find simple way to filter by user (who doesn't want to receive notifications), but default settings (notify on change status) are also viable.